drivers/bridge/bridge.go

```diff
@@ -1009,6 +1012,10 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		}
 	}
 
+	if err := d.storeUpdate(endpoint); err != nil {
+		logrus.Warnf("Failed to save bridge endpoint %s to store: %v", ep.id[0:7], err)
```

Comment:
Is there a problem in failing the operation if the `storeUpdate` fails ?
I think that will be consistent with other store failures in the create path.
(Its understandable that we can warn on Delete - since there is no recovery path).